docs/source/en/api/pipelines/animatediff.md

```diff
@@ -422,6 +422,18 @@ Make sure to check out the Schedulers [guide](../../using-diffusers/schedulers)
     - enable_vae_tiling
     - disable_vae_tiling
 
+## AnimateDiffVideoToVideoPipeline
+
+[[autodoc]] AnimateDiffVideoToVideoPipeline
+  - all
+  - __call__
+    - enable_freeu
+    - disable_freeu
+    - enable_vae_slicing
+    - disable_vae_slicing
+    - enable_vae_tiling
+    - disable_vae_tiling
```

Comment:
I think these can all be removed since `-all` includes them (if you can make the same changes to `AnimateDiffPipeline`, that'd be awesome 🙏 )

```suggestion
```